native/SampleApps/MobileSyncExplorer/MobileSyncExplorer/Classes/AppDelegate.m

```diff
@@ -184,10 +184,4 @@ - (void)resetViewState:(void (^)(void))postResetBlock
     }
 }
 
-- (void)sdkManagerWillResignActive {
```

Comment:
I tried locking/unlocking with the keyboard up and snapshot enabled and didn't notice any issues without this, let me know if there's a more specific scenario I should test